source/_posts/2015-09-27-selecting-apprentices.markdown

```diff
@@ -0,0 +1,59 @@
+---
+layout: post
+title: "Selecting Apprentices"
+date: 2015-09-27 14:56:40 -0600
+comments: true
+categories: interviewing hiring apprenticeship mentoring ruby
+---
+
+> This is the second post on my experience running the [MX ruby apprenticeship program](http://www.mx.com/company/apprenticeships/).
+The [Previous Post](/blog/2015/09/09/hiring-apprentices/) focused on our goals and plans for the apprenticeship.
+
+With dates planned it was time to get some applications for the apprenticeship.
+How should someone apply to the apprenticeship?
+How would we know which candidates best fit our criteria of being self-motivated to learn new things and tackle hard problems?
+Who would bother applying in the first place?
+
+## The Application
+
+Since we had a pretty ambiguous selection criteria, we decided to use a similarly ambiguous application medium.
+
+> Make us a page, anywhere on the internet, that provides evidence that you are self-motivated, learn new things easily and like to solve complex problems.
+
+The requirement of making a page helps to enforce our base criteria for what skills an apprentice should have coming into the program.
+But it also lets the apprentices have a lot of freedom in choosing how to present themselves.
+They could use videos, animations, gifs, etc.
+<!--more-->
+
+We ended up receiving 55 applications and they were as different from each other as we were hoping.
+Here are a few that caught our attention:
+
+* [Danielle Hunter](http://danhunter.github.io/) created a page that was compelling both visually and as a narrative (protip: read the recommendation at the bottom).
+* [Aristides Perez](http://aperezmontan.github.io/) came out of the DevBootcamp program and had a good overview of some of this projects.
+* [Jean Simon](http://www.jeansimon.me/mx) is an excellent of example of keeping it very simple. There is very little visually happening, but the writing was compelling to us.
+* [Cam Kidman](http://cam.thefitdeveloper.com/) is one of the only examples I've seen where calling out your strengths *and weaknesses* came off really well.
+
+## Evaluation
+
+Given such a diverse set of applications we needed a way to evaluate them to pick a set of people to bring back for interviews.
+I made a github repo called `apprenticeship-applications` and created an issue for each applicant that contained a link to their application.
+The entire MX engineering team pitched in to spend a few minutes evaluating different candidates.
+Team members would notice different things and add comments about the applicants' github history, application page and even twitter accounts.
+Eventually one person needed to see them all so that the judging could be done on a single scale.
+So after all the other comments I went through and scored each of the 55 applications and picked 15 people to bring in for interviews.
```

Comment:
Maybe include a little more detail on how you scored them, what you liked seeing and what you weren't very impressed with, and maybe whether or not there was anything in common between applications that you chose